### PR TITLE
feat: require draft revisions in ipr disclosures. Explain missing ones.

### LINKED ISSUE #6155: Require revisions on draft specific IPR disclosures
Since ~2014, the IPR disclosure app has not required disclosers to specify specific revisions of a draft to which a disclosure pertains. Change the form to require that draft revisions be specified, and change the display of existing disclosures that do not specify a revision to note what the revision was at the time the disclosure was posted and explain that we do not know what the intent of the discloser was as to which revisions the disclosure pertained.

### PR DESCRIPTION
**Pull request by rjsparks**
Fixes #6155

### CODE REVIEW

**Review by jennifer-richards — APPROVED** [review]
Approved! (A couple style nits and a possible (but not required) refactor noted inline.)

File: ietf/ipr/templatetags/ipr_filters.py [hunk 1]
```diff
@@ -26,3 +30,37 @@ def render_message_for_history(msg):
...
+
+def draft_rev_at_time(iprdocrel):
+    draft = iprdocrel.document.document
+    event = iprdocrel.disclosure.get_latest_event_posted()
+    if event is None:
+        return ("","The draft's revision at the time this disclosure was posted could not be determined.")
+    time = event.time
+    if not NewRevisionDocEvent.objects.filter(doc=draft).exists():
+        return ("","The draft's revision at the time this disclosure was posted could not be determined.")
+    revs_before = NewRevisionDocEvent.objects.filter(doc=draft, time__lte=time).order_by('-time')
```
[the remark this answers is not in the document]
Comment (larseggert) [reply ietf/ipr/templatetags/ipr_filters.py:42]: draft -> Internet-Draft (per guidance from @ghwood)

**Review by larseggert — CHANGES_REQUESTED** [review]
draft -> Internet-Draft everywhere

**Comment by rjsparks** [comment]
This also needs to test another facet - commit incoming

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**